src/core/initialize.js

```diff
@@ -0,0 +1,171 @@
+const fs = require("fs");
+const path = require("path");
+const inquirer = require("inquirer");
+const Generate = require("../cli/generate");
+const logger = require("../utils/logger");
+const {getPackageJson} = require("../utils/fs");
+const Telemetry = require("../utils/telemetry");
+const Locale = require("../locales")();
+const Config = require("../config");
+const Executor = require('./executor')
+
+async function initialize(program = {}) {
+  const { folder } = program
+
+  let answers = {
+    name: "app",
+    port: 8000,
+    description: "Configuration generated by restqa init",
+    telemetry: true
+  };
+
+  const questionName = {
+    type: "input",
+    name: "name",
+    default: answers.name,
+    message: Locale.get("service.init.questions.name")
+  }
+
+  const questionPort = {
+    type: "input",
+    name: "port",
+    message: Locale.get("service.init.questions.port"),
+    default: answers.port
+  }
+
+  const questionCommand = {
+    type: "input",
+    name: "command",
+    message: Locale.get("service.init.questions.command"),
+  }
+
+  const questionTelemetry = {
+    type: "confirm",
+    name: "telemetry",
+    message: Locale.get("service.init.questions.telemetry"),
+    default: answers.telemetry
+  }
+
+  const questions = [
+    questionName,
+    questionPort,
+    questionCommand,
+    questionTelemetry
+  ]
+
+  const packageJson = getPackageJson(folder);
+  if (packageJson) {
+    questions.shift()
+    const availablesCommands = Object.keys(packageJson.scripts)
+    questionCommand.type = 'list'
+    questionCommand.choices = availablesCommands
+      .filter(cmd => {
+        return cmd !== "test"
+      })
+      .map(cmd => {
+        if (cmd !== 'start') {
```

Comment:
Actually what is happening here is:
1. if the package.json exist we are checking the command available to know which one to propose in a list to user
2. Then we exclude the "test" from the list since we know the test script does not run the microservice
3. Since we know only `npm start` is not running like `npm run start` we are just reshaping all the other command to be fomated like `npm run cmd`